.storybook/stories/KivaClassicBasicLoanCard.stories.js

```diff
@@ -29,6 +29,11 @@ const queryResult = {
 				status: "fundraising",
 				loanAmount: "250000.00",
 				borrowerCount: 1,
+				fundraisingPercent: 0.75,
+				unreservedAmount: '600',
+				// This doesn't seem to be doing anything, are we able to use the
+				// timeLeftMixin within storybook?
+				timeLeft: '10000',
```

Comment:
I don't think you need this here as there is no loan field for `timeLeft`.

`timeLeftMessage` and `expiringSoonMessage` are exposed from the mixin and rely on `plannedExpirationDate`.